src/Infrastructure/LeanCode.ExternalIdentityProviders/Facebook/FacebookUser.cs

```diff
@@ -7,14 +7,22 @@ public sealed class FacebookUser
         public string? FirstName { get; }
         public string? LastName { get; }
         public string Photo { get; }
+        public bool IsSilhouette { get; }
 
-        public FacebookUser(string id, string? email, string? firstName, string? lastName, string photo)
+        public FacebookUser(
```

Comment:
By the way, why isn't this a `record` like other user types?

Reply:
I would guess that this is leftover after FacebookModule 🕯️

Reply:
I will make it consistent in a separate PR (it needs some more work).